### DISCUSSION TITLE
Opera and GIA

### DISCUSSION THREAD

**Discussion by Cessnafan152** [Ascoware/get-iplayer-automator#350] [General]
Hi, is it possible to include Opera in the choice of browsers? I note you have Edge now included, however I can't get that to run on my Mac! Many thanks, Mike

**Comment by skovatch** [comment]
Opera now supports AppleScript but doesn't allow AppleScript to read the HTML on the page, just the URL. That should be okay at the expense of missing some metadata up front.

**Comment by Cessnafan152** [comment]
Hi Scott

Thanks! Got there in the end. A great app, by the way!

Cheers

Mike

> On 17 Aug 2021, at 05:07, Scott Kovatch ***@***.***> wrote:
> 
> 
> Opera now supports AppleScript but doesn't allow AppleScript to read the HTML on the page, just the URL. That should be okay at the expense of missing some metadata up front.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/Ascoware/get-iplayer-automator/discussions/350#discussioncomment-1191870>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AREUITNWRU5SSI4XJT3T5CTT5FSIFANCNFSM5CFVRA2A>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>.
> 

Mike Didsbury, Secretary
